content/docs/user-guide/components/reference/atom/debug-rendering.md

```diff
@@ -0,0 +1,91 @@
+| **Debug Light Elevation** | Determines the elevation in degrees used to determine the position of the debug directional light (change this to move the directional light up/down). | 60.0 |
+| **Debug Light Color** | The color of the debug directional light. | White |
+| **Debug Light Intensity** | The intensity of the debug directional light. | 2.0 |
+
+
+## Material Override Properties
+
+The Debug Render Component allows users to override material values for all materials in the scene. This gives users extra options which which to vizualize and debug lighting and materials in the scene. For example, one use case would be to override the color of all objects in the scene to white or grey in order to get a better sense of the lighting in the scene.
+
+Values that can be overriden are Base Color, Roughness and Metallic. Normals maps and detail normal maps cannot be overriden but they can be disabled (if both are disabled then materials in the scene will use vertex normals only).
+
+| Property | Description | Default Value |
+|-|-|-|
+| **Override Base Color** | Whether to override the base color for all materials in the scene to the value specified below. | Off |
+| **Base Color Value** | The value used to override base color on all materials in the scene. For example you can set this to red to see what your scene would look like if it were entirely red. | RGB (128, 128, 128) |
+| **Override Roughness** | Whether to override the roughness for all materials in the scene to the value specified below. | Off |
+| **Roughness Value** | The value used to override roughness on all materials in the scene. For example you can set this to zero if you want to visualize a very glossy and reflective scene. | 1.0 |
+| **Override Metallic** | Whether to override the metallic for all materials in the scene to the value specified below. | Off |
+| **Metallic Value** | The value used to override metallic on all materials in the scene. For example you can set this one to make all materials in your scene metals. | 0.0 |
+| **Enable Normal Maps** | Whether to enable/disable normal maps on all materials in the scene (note that disabling normal maps will also disable detail normal maps, and thus only the vertex normal will be used in shading calculations). | On |
+| **Enable Detail Normal Maps** | Whether to enable/disable detail normal maps on all materials in the scene (note that not all materials provide or support detail normal maps, therefore those materials will be unaffected by this option). | On |
+
+
+## Custom Debug Properties
+
+When writing shaders, it can be useful to have immediate access to a tweakable value for debug purposes. The Debug Render Component provides access to **four debug booleans** and **four debug floats** that are passed into the Scene Shader Resource Group. These values are accessible from any shader that includes the Scene SRG.
```

Comment:
```suggestion
When debugging shaders, it can be useful to have immediate access to tweakable values. The Debug Render component provides access to four _debug booleans_ and four _debug floats_. These values are passed into the Scene Shader Resource Group (SRG) and are accessible from any shader that includes `SceneSRG`. For more information on SRGs, see [Shader Resource Groups](/docs/atom-guide/dev-guide/shaders/azsl/#shader-resource-groups).
```